components/lvgl_esp32_drivers/lvgl_tft/rm67162Lilygo.c

```diff
@@ -173,9 +173,9 @@ void lcd_init(void)
 
     //Reset the display
 	gpio_set_level(TFT_QSPI_RST, 0);
-	vTaskDelay(300 / portTICK_RATE_MS);//default 100ms
+	vTaskDelay(300 / portTICK_PERIOD_MS);//default 100ms
 	gpio_set_level(TFT_QSPI_RST, 1);
-	vTaskDelay(300 / portTICK_RATE_MS);//default 100ms
+	vTaskDelay(300 / portTICK_PERIOD_MS);//default 100ms
```

Comment:
you should not change any part of the code out of your task scope

Reply:
because my project did not compile !